src/pages/ReimbursementAccount/BankAccountStep.js

```diff
@@ -80,6 +79,16 @@ class BankAccountStep extends React.Component {
         return errors[inputKey] ? this.props.translate(this.errorTranslationKeys[inputKey]) : '';
     }
 
+    setSubStep(subStep) {
+        console.debug('setSubStep', subStep);
```

Comment:
If we are planning to leave this in we might want to give it a better text description like 

`[BankAccountStep] sub step selected: ${subStep}`